packages/nimble-components/src/table/components/header/template.ts

```diff
@@ -44,5 +47,9 @@ export const template = html<TableHeader>`
                 ></${iconTwoSquaresInBracketsTag}>
             `)}
         `)}
+
+        ${when(x => x.alignment === TableColumnAlignment.right, html`
```

Comment:
Am I correct in saying that no matrix test has been updated after your change? If not, should there be a matrix test that exercises this use-case?

Reply:
The number-text column matrix tests [did have a change](https://www.chromatic.com/pullrequest?appId=60e89457a987cf003efc0a5b&number=2245&activeElementId=comparison-test-key-64d2b2b73a35b8c89d2fe96d-1200px&view=changes)